mcapi/src/internal/java/net/flintmc/mcapi/internal/world/codec/DefaultDatapackCodecRepository.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * FlintMC
+ * Copyright (C) 2020-2021 LabyMedia GmbH and contributors
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package net.flintmc.mcapi.internal.world.codec;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Maps;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import java.util.Map;
+import net.flintmc.framework.inject.implement.Implement;
+import net.flintmc.mcapi.world.codec.DatapackCodec;
+import net.flintmc.mcapi.world.codec.DatapackCodec.Factory;
+import net.flintmc.mcapi.world.codec.DatapackCodecRepository;
+
+@Singleton
+@Implement(DatapackCodecRepository.class)
+public class DefaultDatapackCodecRepository implements DatapackCodecRepository {
+
+  private final Map<String, DatapackCodec> datapackCodecs;
+
+  @Inject
+  private DefaultDatapackCodecRepository(Factory datapackCodecFactory) {
+    this.datapackCodecs = Maps.newConcurrentMap();
+    this.register(
+        "vanilla", datapackCodecFactory.create(ImmutableList.of("vanilla"), ImmutableList.of()));
```

Comment:
Iirc the google style has a higher limit on the width of the line, so this should be on one line (just apply the style as mentioned above)